src/components/grid-aware/Footer/InfoBlock/InfoBlock.module.css

```diff
@@ -0,0 +1,44 @@
+/* Background bleed */
```

Comment:
I have a couple styling observations for the InfoBlock, but I'm going to leave them at the top of this file, since there's no other good place to attach it.

1. You need to override the `<address>` tag's default styles, specifically the fact that it appears to be italicized by default
2. You need to override the padding of the info block in mobile, which is smaller in order to allow the address to fit in one row (at least on a 375px-wide device).

Reply:
ok